### DISCUSSION TITLE
How does Supabase count the number of emails sent for signup and api call for Authentication.

### DISCUSSION THREAD

**Discussion by Prateek32177** [supabase/supabase#29167] [Questions] [answered]
How does Supabase count the number of emails sent for signup auth confirmation, as it's shown that 30 is rate limit per hour but rate limit exceed after 3-4 signup confirmation been sent, please help? Also how auth api call gets counted as I am seeing auth signin performed once or twice in a day but counts are in hundreds?###

**Comment by GaryAustin1** [comment]
There may be a bug on the 30 showing for some users or condition.  There is an open issue on that.  
But the limit is 3 (including errors) with the built in SMTP.  Either test locally or get an SMTP email provider.  There are many with decent free plans.

**Reply by Prateek32177** [reply]
Yeah exactly 3 times only it allows to send the mail while 30 is mentioned in rate limit as u said it might be an bug, Can you please suggest me any SMTP email provider which works well with Supabase and have initial good free plan?

**Reply by GaryAustin1** [reply]
Resend is tightly integrated with Supabase.  I use Brevo which has a good free plan.  I've seen 1/2 dozen others mentioned though that all work.

**Reply by Prateek32177** [reply]
Thanks alot @GaryAustin1 , that helps , do you have any info on my second query , like how does Supabase calculate the api call for Authentication users as I have seen that even I use to login few times a day < 10 times ,  Supabase log shows 100s of call , is it something I am making mistake like might be due to rerender or Supabase have different approach to calculate ?

**Reply by GaryAustin1** [reply]
Without knowing the API calls it is hard to say.  There is a refresh jwt token call every hour by default if a client is "on".  There are also API calls by the dashboard constantly while it is "on".

**Reply by GaryAustin1** [reply, marked as answer]
An update here on the 30 limit "error" and that it should be 6 emails an hour now.  https://github.com/supabase/supabase/issues/26623#issuecomment-2338901914
